.travis.yml

```diff
@@ -72,14 +72,20 @@ matrix:
         - python: 3.6
           env: NUMPY_VERSION=prerelease
 
+        # pycodestyle test, see setup.cfg for tested inconsistencies
+        - python: 3.6
+          env: MAIN_CMD='pycodestyle ccdproc --count  --show-source --show-pep8'
+               SETUP_CMD=''
+               CONDA_DEPENDENCIES='pycodestyle'
```

Comment:
you don't need this, ``ci-helpers`` checks for ``'pycodestyle'`` in ``MAIN_CMD`` and installs it when needed.